src/js/view/controls/components/tooltip.js

```diff
@@ -5,30 +5,16 @@ define([
 ], function(Events, utils, _) {
 
     return class Tooltip {
-        constructor(name, ariaText, ariaShown, elementShown) {
+        constructor(name, ariaText, elementShown) {
```

Comment:
`ariaShown` must have been passed in as `false` from somewhere (Menu?) but I don't see the change in signature addressed anywhere but `VolumeTooltip`. It seems like you haven't updated all places where this constructor is run, unless these were undefined.